static/scss/audio-player.scss

```diff
@@ -0,0 +1,269 @@
+.audio-player-container-outer.hidden {
+  display: none;
+}
+
+.audio-player-container-outer {
+  background-color: white;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  box-shadow: 0px -4px 15px rgba(50, 50, 50, 0.3);
+  font-family: Helvetica, sans-serif;
+  left: 0;
+  bottom: 0;
+  width: 100vw;
+
+  @include breakpoint(desktop) {
+    height: 98px;
+  }
+  @include breakpoint(mobile) {
+    height: 128px;
+  }
+
+  .audio-player-container-inner {
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex: 1;
+
+    @include breakpoint(desktop) {
+      padding: 8px 12px;
+      height: calc(100% - 16px);
+      width: calc(100% - 24px);
+    }
+    @include breakpoint(mobile) {
+      flex-direction: column;
+      padding: 8px 12px;
+      height: calc(100% - 16px);
+      width: calc(100% - 24px);
```

Comment:
is the `24px` here because it's 2x the `12px` in the padding? if so might be good to make these things variables so it's clear how they relate